darts/tests/test_timeseries.py

```diff
@@ -89,6 +89,44 @@ def test_pandas_creation(self):
         ts_pd_df = ts.pd_dataframe()
         assert ts_pd_df.equals(pd_df)
 
+    def test_pandas_group_creation(self):
```

Comment:
Instead of this tests, you could just add another check to `test_timeseries_from_longitudinal_df()` in `test_timeseries_static_covariates.py`.

E.g. after`ts_groups7`, you can generate `ts_groups7_parallel` with `n_jobs=-1` and check that the output is identical to `ts_groups7`